ci/travis_script_python.sh

```diff
@@ -47,6 +47,7 @@ fi
 
 conda create -y -q -p $CONDA_ENV_DIR \
       --file $TRAVIS_BUILD_DIR/ci/conda_env_python.yml \
+      nomkl \
```

Comment:
We should add it to the python docker image as well.

Reply:
done